src/main/java/org/isf/lab/dto/LaboratoryDTO.java

```diff
@@ -63,15 +64,26 @@ public class LaboratoryDTO {
     private String patName;
 
     @ApiModelProperty(notes = "Laboratory Patient InOut", example = "0", position = 10)
-    private String inOutPatient;
+    private PatientSTATUS inOutPatient;
 
     @ApiModelProperty(notes = "Laboratory Patient Age", position = 11)
     private Integer age;
 
     @ApiModelProperty(notes = "Laboratory Patient Sex", example = "M", position = 12)
     private String sex;
+    
+    @ApiModelProperty(notes = "Laboratory status", example = "Draft", position = 13)
+    private LaboratorySTATUS status;
+
+    public LaboratorySTATUS getStatus() {
```

Comment:
Why is the variable `LaboratorySTATUS` not `LaboratoryStatus`?   Using uppercase like this seems wrong.